enterprise/kernel/src/main/java/org/neo4j/kernel/impl/store/format/highlimit/RelationshipGroupRecordFormat.java

```diff
@@ -181,7 +188,8 @@ private void readFixedReferencesMethod( RelationshipGroupRecord record, PageCurs
         // [   x,    ] high owner bits
         long modifiers = cursor.getByte();
 
-        int type = cursor.getShort() & 0xFFFF;
+        int typeLowWord = cursor.getShort() & 0xFFFF;
+        int type = (((cursor.getByte() & 0xFF) << Short.SIZE) | typeLowWord);
```

Comment:
would be nice with a utility method for reading a 3-byte int off of a PageCursor.